optional patch to make my noname h6 tv box work
see: https://freenode.irclog.whitequark.org/linux-sunxi/2020-08-11

only the rank=1 part is really needed - looks like there was some problem with earlier testing
see: https://freenode.irclog.whitequark.org/linux-sunxi/2020-08-28#27817273

diff --git a/arch/arm/mach-sunxi/dram_sun50i_h6.c b/arch/arm/mach-sunxi/dram_sun50i_h6.c
index 9e34da4747..f392312598 100644
--- a/arch/arm/mach-sunxi/dram_sun50i_h6.c
+++ b/arch/arm/mach-sunxi/dram_sun50i_h6.c
@@ -652,7 +654,7 @@ unsigned long sunxi_dram_init(void)
 			(struct sunxi_mctl_com_reg *)SUNXI_DRAM_COM_BASE;
 	struct dram_para para = {
 		.clk = CONFIG_DRAM_CLK,
-		.ranks = 2,
+		.ranks = 1,
 		.cols = 11,
 		.rows = 14,
 		.bus_full_width = 1,
